ui/components/multichain/pages/connections/index.js

```diff
@@ -0,0 +1 @@
+export { Connections } from './connections';
```

Comment:
We can remove this layer since there's only 1 `connecions` file ?

Reply:
Aah! we need it since this component is gonna have multiple child components. Main agenda of this PR is just to set up the connections route 😅